src/runner.py

```diff
@@ -499,3 +494,31 @@ def _execute(self, cmd: list[str], cwd: Optional[str] = None, **kwargs) -> str:
             raise RunnerExecutionError(
                 f"Failed to execute command in {self.config.name}: {cmd}"
             ) from err
+
+    def _apt_install(self, packages: Iterable[str]) -> None:
+        """Installs the given APT packages.
+
+        This is a temporary solution to provide tools not offered by the base ubuntu image. Custom
+        images based on the GitHub action runner image will be used in the future.
+
+        Args:
+            packages: Packages to be install via apt.
+        """
+        self._execute(["/usr/bin/apt", "update"])
```

Comment:
Is it correct to use `apt` here instead of `apt-get` ? The former is intended as an end user interface (and is more subject to changes).  
This comment applies for the other invocations of `apt`

Reply:
Updated the code to change `apt` to `apt-get`.